### PR TITLE
Fix: tag multilang url and spaces in url

### PR DESCRIPTION
**Pull request by maxiroellplenty**
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been documented
- [x] Changes have been tested by the author
- [x] Changes have been tested by the reviewer

@plentymarkets/plentyshop

### CODE REVIEW

File: meta/documents/changelog_de.md [hunk 1]
```diff
@@ -2,10 +2,24 @@
...
+### ToDo
+
+- Aufgrund von Änderungen an bestehenden Widgets müssen die ShopBuilder-Inhalte im Menü **CMS » ShopBuilder** über die Schaltfläche **Inhalte neu generieren** aktualisiert werden.
+
 ### Hinzugefügt
 
 - Im Assistenten kann nun die Platzhalter-Zahlungsart "Bereits bezahlt" für Bestellungen mit einer Warenkorbsumme von null aktiviert werden.
 
+### Behoben
+- Die Tag Url berücksichtigt jetzt die ausgewählte Shop-Sprache.
```
Comment (Tim-M-S) [inline meta/documents/changelog_de.md]:
```suggestion
- Tag-URLs berücksichtigen jetzt die Shop-Sprache.
```

File: meta/documents/changelog_de.md [hunk 2]
```diff
@@ -2,10 +2,24 @@
...
+- Die Tag Url enthält nun keine Leerzeichen mehr.
```
Comment (Tim-M-S) [inline meta/documents/changelog_de.md]:
```suggestion
- Tag-URLs die aus mehreren Wörtern bestehen enthalten nun keine Leerzeichen mehr.
```

File: meta/documents/changelog_en.md [hunk 3]
```diff
@@ -2,10 +2,24 @@
...
+
+- The ShopBuilder contents for order confirmation pages must be updated in the **CMS » ShopBuilder** menu via the **Regenerate contents** button.
+
 ### Added
 
 - In the assistant, the placeholder payment method "Already paid" can now be activated for orders with a shopping cart total of zero.
 
+### Fixed
+
+- The tag url now takes the selected shop language into account.
```
Comment (Tim-M-S) [inline meta/documents/changelog_en.md]:
```suggestion
- Tag URLs now consider the store language.
```

File: meta/documents/changelog_en.md [hunk 4]
```diff
@@ -2,10 +2,24 @@
...
+- The tag url no longer contains spaces.
```
Comment (Tim-M-S) [inline meta/documents/changelog_en.md]:
```suggestion
- Tag URLs that consist of multiple words no longer contain spaces.
```

File: meta/documents/changelog_de.md [hunk 5]
```diff
@@ -2,10 +2,24 @@
...
+
+- Aufgrund von Änderungen an bestehenden Widgets müssen die ShopBuilder-Inhalte im Menü **CMS » ShopBuilder** über die Schaltfläche **Inhalte neu generieren** aktualisiert werden.
+
 ### Hinzugefügt
 
 - Im Assistenten kann nun die Platzhalter-Zahlungsart "Bereits bezahlt" für Bestellungen mit einer Warenkorbsumme von null aktiviert werden.
 
+### Behoben
+- Tag-URLs berücksichtigen jetzt die Shop-Sprache.
+- Tag-URLs die aus mehreren Wörtern bestehen enthalten nun keine Leerzeichen mehr.
```
Comment (Tim-M-S) [inline meta/documents/changelog_de.md]:
```suggestion
- Tag-URLs, die aus mehreren Wörtern bestehen, enthalten keine Leerzeichen mehr.
```

**Review by Tim-M-S — DISMISSED** [review]

**Comment by maxiroellplenty** [comment]
testing done ✅

**Review by Tim-M-S — APPROVED** [review]

**Pull request merged**